packages/js/src/executors/tsc/schema.json

```diff
@@ -61,6 +61,31 @@
       "description": "When `updateBuildableProjectDepsInPackageJson` is `true`, this adds dependencies to either `peerDependencies` or `dependencies`.",
       "enum": ["dependencies", "peerDependencies"],
       "default": "peerDependencies"
+    },
+    "external": {
+      "oneOf": [
+        {
+          "type": "string",
+          "description": "Mark which other projects to be external to the project. Eg: projects need to be published and installed separately",
+          "enum": ["all", "none"],
+          "default": "all"
+        },
+        {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Mark which other projects to be external to the project. Eg: projects need to be published and installed separately"
```

Comment:
I think this description isn't needed, or it should say something like `A list projects to be treated as external.`.